cmd/apply.go

```diff
@@ -86,7 +89,15 @@ func apply(c *cobra.Command, _ []string) error {
 		}
 	}
 
-	numAffectedRows, err := client.ApplyDMLFile(ctx, dml, partitioned)
+	p, err := priorityTypeOf(priority)
+	if err != nil {
```

Comment:
According to the doc: https://pkg.go.dev/google.golang.org/genproto/googleapis/spanner/v1#RequestOptions_Priority
If the priority not set, `RequestOptions_PRIORITY_UNSPECIFIED` will be used.

So, I think it's better to use `RequestOptions_PRIORITY_UNSPECIFIED` as default rather than `high`.

Reply:
As a command line option, I think it would be easier to understand if the default is `high` rather than `unspecified`.
`priorityTypeOf` would be better to handle it as `unspecified` internally.

Reply:
I didn't mean that we should make `--priority` option have `unspecified`. Instead, I think when users don't specify `--priority`, it should behave like `RequestOptions_PRIORITY_UNSPECIFIED`.

Reply:
I've made some changes, PTAL 🙏